scripts/action-recognition/train_recognizer.py

```diff
@@ -256,7 +264,11 @@ def main():
         lr_decay_epoch = [int(i) for i in opt.lr_decay_epoch.split(',')]
     lr_decay_epoch = [e - opt.warmup_epochs for e in lr_decay_epoch]
 
-    optimizer = 'sgd'
+    if opt.slowfast:
```

Comment:
Just curious, how large is the difference if we use 'nag' or 'sgd' for all action-recognition models?

Reply:
Then we may use `nag` for all models

Reply:
Yes, we could. But right now, I haven't test all the models with `nag`. For example, all the 2D TSN models. I can rerun all the models in the future to see if I can use `nag` for everything.